zipline/testing/fixtures.py

```diff
@@ -884,9 +909,13 @@ def make_adjustment_writer_daily_bar_reader(cls):
         return cls.bcolz_daily_bar_reader
 
     @classmethod
+    def make_adjustment_db_conn_str(cls):
+        return ':memory:'
```

Comment:
Oh, this is going straight to sqlite3.connect instead of sqlalchemy.  Might be worth a comment.